cli/cli.go

```diff
@@ -94,6 +101,11 @@ func registerFlags(c *Config, f *flag.FlagSet) {
 	f.StringVar(&c.IP, "ip", "", "remote server ip")
 	f.StringVar(&c.Remote, "remote", "", "remote CFSSL server")
 	f.StringVar(&c.Label, "label", "", "key label to use in remote CFSSL server")
+	f.StringVar(&c.ResponderFile, "responder", "", "Certificate for OCSP responder")
+	f.StringVar(&c.Status, "status", "good", "Status of the certificate: good, revoked, unknown")
+	f.IntVar(&c.Reason, "reason", 0, "Reason code for revocation")
+	f.StringVar(&c.RevokedAt, "revoked-at", "now", "Date of revocation (YYYY-MM-DD)")
+	f.IntVar(&c.Interval, "interval", 3600*24*4, "Interval between OCSP updates, in seconds (default: 4 days)")
```

Comment:
Changed to "4 * helpers.OneDay", which seems like it strikes the right balance between readability and constant specificity.